app/src/main/java/com/internship/move/feature/onboarding/OnboardingFragment.kt

```diff
@@ -34,7 +42,7 @@ class OnboardingFragment : Fragment(R.layout.fragment_onboarding) {
                 }
                 binding.onboardingVP2.currentItem = binding.onboardingVP2.currentItem + 1
             } else {
-                initPersistence()
+                scope.launch { viewModel.logIn() }
```

Comment:
Try something new: Have a liveData that holds a boolean value that indicates if the user's logged in and observe it.